manager/eris-mint/evm/vm.go

```diff
@@ -562,25 +564,32 @@ func (vm *VM) call(caller, callee *Account, code, input []byte, value int64, gas
 
 		case EXTCODESIZE: // 0x3B
 			addr := stack.Pop()
+
 			if useGasNegative(gas, GasGetAccount, &err) {
 				return nil, err
 			}
 			acc := vm.appState.GetAccount(addr)
 			if acc == nil {
-				return nil, firstErr(err, ErrUnknownAddress)
+				if _, ok := registeredNativeContracts[addr]; !ok {
```

Comment:
Please remove this change from the PR since it is not related to https://github.com/eris-ltd/eris-db/issues/417